lisa/core/sshConnection.py

```diff
@@ -1,4 +1,9 @@
-import os
+from typing import Optional
+
+import paramiko
```

Comment:
See comment below on considering ssh2-python for performance reasons over paramiko.

Reply:
It looks a lower level protocol wrapper, and may need more code to get it works. in backlog.
`Improve ssh, and validate the ssh connection can be reconnected, once it's broken by some reasons.`